providers/profitbricks/src/main/java/org/jclouds/profitbricks/compute/config/ProfitBricksComputeServiceContextModule.java

```diff
@@ -132,11 +135,11 @@ ProvisioningManager provideProvisioningManager(Closer closer) {
 
    @Provides
    @Singleton
-   @Named(POLL_PREDICATE_SNAPSHOT)
-   Predicate<String> provideSnapshotAvailablePredicate(final ProfitBricksApi api, ComputeConstants constants) {
+   @Named(TIMEOUT_SNAPSHOT_AVAILABLE)
```

Comment:
Better use the existing `ComputeServiceProperties.TIMEOUT_IMAGE_AVAILABLE` property.